instagram/src/components/CommentSection/CommentContainer.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Comment from './Comment';
+import Timestamp from './Timestamp';
+import CommentInput from './CommentInput';
+
+class CommentContainer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: props.comments,
+            newComment: "",
+            likes: 0
+        };
+    }
+
+    addNewComment = event => {
+        event.preventDefault();
+            
+        this.setState({
+            comments: [
+                ...this.state.comments,
+                {username: "User", text: this.state.newComment}
+            ],
+            newComment: ""
+        });
+        console.log(this.state.comments);
+    }
+
+    handleChanges = event => {
+        this.setState({[event.target.name]: event.target.value})
+        this.setState({newComment: event.target.value});
```

Comment:
You don't need this line: `this.setState({newComment: event.target.value});`. It's basically a hack since you had `name="comment"` on your input element instead of `name="newComment"`. If you make those changes, you should only be using `this.setState({[event.target.name]: event.target.value})` in the handleChanges function since `event.target.name` evaluates to `newComment`.